package.json

```
@@ -22,6 +22,7 @@
     "test": "jest --config=jest.json",
     "test:ci": "jest --config=jest.json --coverage --runInBand",
     "test:coverage": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "test:report": "jest --config=jest.json --json --outputFile=report.json | exit 0",
```

The pipe-exit-0 call is a little weird

If you're wanting to test:report to unconditionally report success, this is probably what you want:

```suggestion
    "test:report": "jest --config=jest.json --json --outputFile=report.json || true",
```

Some background about the `|| true` idiom @ https://serverfault.com/questions/228281/bash-run-an-executable-giving-a-good-exit-status

OTOH, if you're wanting test:report to report success iff report.json was created -- e.g. if your caller doesn't care if the tests passed or failed, but does care that there is a report -- then you want to append another test:

```suggestion
    "test:report": "jest --config=jest.json --json --outputFile=report.json || true; test -f report.json",
```

---

Separate issue I discovered while testing the above suggestions:

Because of the `"bail": true,` line in `jest.json`, this report won't get created if any tests fail, so a report will only exist if it's going to report 100% success.

I suspect that's not what you're after since that's not a very useful report, so you'd want to force the jest invocation here to override the config file by changing `bail`:

`"test:report": "jest --config=jest.json --bail=false --json ...`

Paired with the above suggestions, maybe something like this:

```suggestion
    "test:report": "jest --config=jest.json --bail=false --json --outputFile=report.json || true; test -f report.json",```